utl/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(flatbuffer)
```

Comment:
this could be removed if `utl/flatbuffer` moves under `vowpalwabbit/parser/flatbuffer`